push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/GCMPushNotificationSender.java

```diff
@@ -81,6 +81,11 @@ public void sendPushMessage(Variant variant, Collection<String> tokens, UnifiedP
         gcmBuilder.addData("alert", message.getAlert());
         gcmBuilder.addData("sound", message.getSound());
         gcmBuilder.addData("badge", "" + message.getBadge());
+        /*
+        The Message defaults to a Normal priority.  High priority is used
+        by GCM to wake up devices in Doze mode as well as apps in AppStandby 
+        mode.  This has no effect on devices older than Android 6.0
+        */
```

Comment:
yep! Abusage might cause trouble for the apps, right? If so, let's add that too :)

more as a warning, that somebody is not defaulting it to HIGH because of stupid reasons